CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# picolibrary-microchip-avr-megaavr
+#
+# Copyright 2020 Andrew Countryman <apcountryman@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+# file except in compliance with the License. You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# File: CMakeLists.txt
+# Description: picolibrary-microchip-avr-megaavr core CMake rules.
+
+cmake_minimum_required( VERSION 3.12.4 )
```

Comment:
Debian Buster currently distributes CMake version 3.13.4.

Reply:
Ubuntu Bionic (used by Travis CI jobs) currently distributes CMake version 3.12.4. Leave as is.